### PR TITLE
fixed some bugs & wordings & topics

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/comtac/CM-2/button_pressed.schema.json [hunk 1]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Button pressed",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```
Comment (mariusgiger) [inline types/comtac/CM-2/button_pressed.schema.json:7]: better call this buttonPressed

File: types/comtac/CM-1/button_pressed.schema.json [hunk 2]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Button pressed",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```
Comment (mariusgiger) [inline types/comtac/CM-1/button_pressed.schema.json:7]: better call this buttonPressed

File: types/comtac/CM-2/event.schema.json [hunk 3]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Event",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
+      "title": "Status",
```
Comment (mariusgiger) [inline types/comtac/CM-2/event.schema.json:8]: better call this buttonPressed

File: types/comtac/CM-2/sleep.schema.json [hunk 4]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Sleep",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
+      "title": "Status",
```
Comment (mariusgiger) [inline types/comtac/CM-2/sleep.schema.json:8]: better call this buttonPressed

File: types/comtac/CM-2/timer.schema.json [hunk 5]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Timer",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```
Comment (mariusgiger) [inline types/comtac/CM-2/timer.schema.json:7]: better call this timedUplink, when is this created?

File: types/comtac/MS-3/event.schema.json [hunk 6]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Event",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```
Comment (mariusgiger) [inline types/comtac/MS-3/event.schema.json:7]: better call this eventUplink

File: types/decentlab/IAM/default.schema.json [hunk 7]
```diff
@@ -3,67 +3,68 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Default",
   "type": "object",
-  "allOf": [
```
Comment (mariusgiger) [inline types/decentlab/IAM/default.schema.json:6]: why this change?

File: types/decentlab/IAM/meta.json [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "IAM",
+  "name": "DL-IAM",
```
Comment (mariusgiger) [inline types/decentlab/IAM/meta.json:2]: please note that name changes are breaking - therefore let's revert this if it is not needed

File: types/miromico/REV2_V5/join.schema.json [hunk 9]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Join",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
+      "title": "Status",
```
Comment (mariusgiger) [inline types/miromico/REV2_V5/join.schema.json:8]: find a better name

File: types/miromico/sod/button_pressed.schema.json [hunk 10]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Button pressed",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```
Comment (mariusgiger) [inline types/miromico/sod/button_pressed.schema.json:7]: find a better name

File: types/miromico/sod/button_pressed_long.schema.json [hunk 11]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Button pressed long",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```
Comment (mariusgiger) [inline types/miromico/sod/button_pressed_long.schema.json:7]: find a better name

File: types/netvox/button/button_pressed.schema.json [hunk 12]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Button pressed",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```
Comment (mariusgiger) [inline types/netvox/button/button_pressed.schema.json:7]: find a better name

File: types/swisscom/multisense-workplace/button_event.schema.json [hunk 13]
```diff
@@ -3,5 +3,11 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Button event",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "status": {
```
Comment (mariusgiger) [inline types/swisscom/multisense-workplace/button_event.schema.json:7]: find a better name

**Review by mariusgiger — CHANGES_REQUESTED** [review]
please review the comments

File: types/comtac/CM-2/timer.schema.json (hunk 5 above)
Comment (J2thatsme) [reply types/comtac/CM-2/timer.schema.json:7]: On a standart send intervals

File: types/decentlab/IAM/default.schema.json (hunk 7 above)
Comment (J2thatsme) [reply types/decentlab/IAM/default.schema.json:6]: You cant create a rule with the decentlab IAM. I figured it cant load the schema so i'am removing it.

File: types/decentlab/IAM/meta.json (hunk 8 above)
Comment (J2thatsme) [reply types/decentlab/IAM/meta.json:2]: Smolo wanted it ^^

File: types/decentlab/IAM/default.schema.json (hunk 7 above)
Comment (mariusgiger) [reply types/decentlab/IAM/default.schema.json:6]: ok get it, this is something we will have to support to allow this schema inheritance

File: types/decentlab/IAM/meta.json (hunk 8 above)
Comment (mariusgiger) [reply types/decentlab/IAM/meta.json:2]: ok call me before merging so that we can coordinate this change

**Review by mariusgiger — APPROVED** [review]
